mtags-interfaces/src/main/java/scala/meta/pc/PresentationCompiler.java

```diff
@@ -95,6 +96,11 @@ public abstract class PresentationCompiler {
      */
     public abstract CompletableFuture<List<TextEdit>> insertInferredType(OffsetParams params);
 
+    /**
+     * Extract method in selected range
+     */
+    public abstract CompletableFuture<List<TextEdit>> extractMethod(OffsetParams params, Range range, Range defnRange);
```

Comment:
This could make the tests a bit easier and no need to forward LSP ranges.
```suggestion
    public abstract CompletableFuture<List<TextEdit>> extractMethod(OffsetParams params, RangeParams range, RangeParams defnRange);
```
Could you also add information